frontend/packages/console-app/src/components/quick-starts/loader/QuickStartsLoader.tsx

```diff
@@ -44,7 +45,10 @@ const QuickStartsLoader: React.FC<QuickStartsLoaderProps> = ({ children }) => {
             />
           );
         })}
-      {children(allowedQuickStarts, loaded)}
+      {children(
+        allowedQuickStarts.filter((qs) => !isDisabledQuickStart(qs)),
+        loaded,
+      )}
```

Comment:
Keep old allowedQuickstarts and add this filter to the `filteredQuickStarts` above. Like:

```
        const filteredQuickStarts = quickStarts.filter(
          (quickstart) => permissionChecks.current[quickstart.metadata.name] && !isDisabledQuickStart(quickstart),
        );
```

Wdyt?

Reply:
this was the earlier implementation though, will discuss and update

Reply:
I've updated `quickStarts` and replaced it with the filtered version